feature-libs/product/image-zoom/components/product-image-zoom/product-image-zoom-dialog/product-image-zoom-dialog.component.html

```diff
@@ -1,4 +1,34 @@
 <div
+  *cxFeature="'a11yZoomDialogCloseOnEsc'"
+  class="cx-image-zoom-dialog"
+  role="dialog"
+  (esc)="close('Escape clicked')"
```

Comment:
It looks like only the keybinding here has been added. I wouldn't consider this a breaking change.

Reply:
Feature flag removed